src/main/scala/tech/cryptonomic/conseil/tezos/Tables.scala

```diff
@@ -204,6 +204,9 @@ trait Tables {
   /** Collection-like TableQuery object for table OperationGroups */
   lazy val OperationGroups = new TableQuery(tag => new OperationGroups(tag))
 
+  val operationGroupsByHash =
```

Comment:
This is a generated file produced using Slick via `GenSchema.scala`. We should move any custom logic out of here or make these changes in the database schema so this functionality is also also auto-generated.

Reply:
Sorry, my bad, didn't notice, I'll take care